DOCS-6688 Fix garbled text (#500)
mulesoft/docs-connectors: modules/ROOT/pages/aggregator/aggregator-examples.adoc

> </aggregators:size-based-aggregator>

Reviewer: INSTRUCTIONS FOR EDIT

Hi Kim, Can you please edit this PR (only the green text--the scope is a bug fix, not a rewrite). When you get to it is fine--not urgent. Thank you!

----

> If you do not want to execute some action for every new element because that would cause too many executions, you can instead use `incremental-aggregation`. For example, you could execute an action every time your list of names increases by an increment of 10% as shown in the below example. You can assume that the logic to be executed is in another flow named `partialAction`. 

Reviewer: Questions about "execute some action":

Did you intend "execute" or "repeat"?
Did you mean "some actions"? "an action"? " "a particular action"?

Reviewer: Also, can the final sentence follow the example, so that we can introduce the example directly?
```suggestion
If you do not want to execute an action for every new element because that would cause too many executions, you can use `incremental-aggregation`. For example, you could execute an action every time your list of names increases by 10%, as shown here:

You can assume that the logic to be executed is in another flow named `partialAction`. 
```

</aggregators:size-based-aggregator>
----

> But what if you do not want to wait forever to fill every batch. In that case, you might want to process all the events that you have even if the `maxSize` is not reached. For this, you can configure a timeout for your aggregator. The below example sets the waiting limit to 10 seconds.

Reviewer: ```suggestion
If you do not want to wait the time it takes to fill every batch, you can process all the events that you have, even if the `maxSize` value is not reached, by setting a timeout for your aggregator. The following example sets the timeout to 10 seconds:
```

----

For this case, the data types that come from the `aggregation-complete` route and `aggregator-listener` are the same. So, for the execution of the flow `whatToDoWithAllTheseNames`, it does not matter where the data comes from.

> However, with this configuration, in the case the aggregation is complete, the flow `whatToDoWithAllTheseNames` will trigger twice: once from the `aggregation-complete` route `flow-ref`, and again from the `flow-ref` in the flow with the `aggregator-listener`.

Reviewer: ```suggestion
However, with this configuration, when the aggregation is complete, the flow `whatToDoWithAllTheseNames` triggers twice: once from the `aggregation-complete` route `flow-ref`, and again from the `flow-ref` in the flow with the `aggregator-listener`.
```

> To avoid multiple executions of the same logic, the `flow-ref` after the `aggregator-listener` should be wrapped in a `choice` component that uses the `isAggregationComplete` attribute to check for an incomplete aggregation. That way, the `flow-ref` is called only when a timeout triggered that flow.

Reviewer: Again suggest splitting para text around the example:
```suggestion
To avoid multiple executions of the same logic, wrap the `flow-ref` value `aggregator-listener` in a `choice` component that uses the `isAggregationComplete` attribute to check for an incomplete aggregation:

Using this syntax, `flow-ref` is called only when a timeout triggers that flow.
```

<flow-ref name="whatToDoWithAllTheseNamesFromSomeCountry"/>
----

> Notice that the example does not configure an eviction time in the aggregator. That is because the default value is `0`, which means the eviction takes place right after the group is released, which is the desired behavior. So if a group completed or timed out, a subsequent value can be added to the group without being rejected.

Reviewer: Is "eviction time" = "timeout"? If so, choose just one term and use it throughout, to avoid confusion.

Reviewer: For nonnative readers, "takes place" > "occurs"
```suggestion
Notice that the example does not configure an eviction time in the aggregator. That is because the default value is `0`, which means the eviction occurs directly after the group is released, which is the desired behavior. Therefore, if a group finishes or is timed out, a subsequent value can be added to the group without being rejected.
```
